setup.md

```diff
@@ -0,0 +1,87 @@
+## Pre-requisites
```

Comment:
Remove this as there is another setup.md under service.

Reply:
done